pkg/controller/apimanager/upgrade_26_to_27.go

```diff
@@ -379,3 +385,29 @@ func (u *Upgrade26_to_27) upgradeImageStreams() (reconcile.Result, error) {
 
 	return reconcile.Result{Requeue: anImageStreamChanged}, nil
 }
+
+func (u *Upgrade26_to_27) upgradeSystemRollingUpdatesCfgMap() (reconcile.Result, error) {
+	cfgMapName := "system"
+	cfgMapEntry := "rolling_updates.yml"
+	expectedValue := "api_as_product: true"
+
+	systemCfgMap := &v1.ConfigMap{}
+	err := u.client.Get(context.TODO(), types.NamespacedName{Name: cfgMapName, Namespace: u.cr.Namespace}, systemCfgMap)
+	if err != nil {
+		return reconcile.Result{}, err
+	}
+
+	var rollingUpdatesTxt string
+	rollingUpdatesTxt, ok := systemCfgMap.Data[cfgMapEntry]
+	if !ok {
+		return reconcile.Result{}, fmt.Errorf("Expected %s key in %s ConfigMap not found", cfgMapEntry, cfgMapName)
+	}
+
+	if !strings.Contains(rollingUpdatesTxt, expectedValue) {
```

Comment:
should we parse yaml?? what about if other key or value contains `expectedValue`??

Reply:
Expected value is key + value already so if there's a match we have what we need I guess.
Probably it would be a good idea to parse YAML yes.